Arm/Memory/SeparateAutomation.lean

```diff
@@ -427,10 +444,18 @@ def simpAndIntroDef (name : String) (hdefVal : Expr) : SimpMemM FVarId  := do
 
 /-- SimpMemM's omega invoker -/
 def omega : SimpMemM Unit := do
-  -- https://leanprover.zulipchat.com/#narrow/stream/326056-ICERM22-after-party/topic/Regression.20tests/near/290131280
-  -- @bollu: TODO: understand what precisely we are recovering from.
-  withoutRecover do
-    evalTactic (← `(tactic| bv_omega))
+  SimpMemM.withMainContext do
+    -- https://leanprover.zulipchat.com/#narrow/stream/326056-ICERM22-after-party/topic/Regression.20tests/near/290131280
+    -- @bollu: TODO: understand what precisely we are recovering from.
```

Comment:
You might want to move this comment down to the `withoutRecover` that it's talking about